env.example

```diff
@@ -49,3 +49,11 @@ DJANGO_SERVER_EMAIL=
 # Sentry
 # Only needed if you want to send errors to Sentry
 DJANGO_SENTRY_DSN=
+
+
+# GUS BUS
+export METACI_RELEASE_WEBHOOK_URL=http://0.0.0.0:8001
+export METACI_RELEASE_WEBHOOK_AUTH_KEY=123645
+export METACI_START_STOP_WEBHOOK=True
+# Only needed when creating implementation steps find from GUS user id
+# export GUS_BUS_OWNER_ID="005B00000000000000"
```

Comment:
@jofsky I meant that we should comment out this entire section starting on line 55. Sending release and start/stop webhooks should be off by default.